_notes/rft.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Request For Teachers
+---
+
+I believe that the RFT should be approached from the "problem" angle. 
+Meaning the teachers should be reaching for a RFT whenever they see a problem.
+Here are some examples off the top of my head:
+
+- Some of my students have a hard time coming up with compelling copywriting for their product launch
```

Comment:
I think its better to start with 
Students currently [present context], but [the gap in context]. Ideally they should be able to [the ideal if the gap is crossed]

e.g Students currently have 3d printers in the homeroom but they lack the practice around CAD tooling. Ideally they are able to print simple everyday items after.